gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Sep 11 16:05:40 EEST 2017
+#Wed Nov 29 11:44:15 CET 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.3-all.zip
```

Comment:
run `./gradlew wrapper` as well to update wrapper bins

Reply:
I did that already, see the change above this file